### DISCUSSION TITLE
WebKit/WebKit repo cannot be analysed

### DISCUSSION THREAD

**Discussion by MainTobias** [github-linguist/linguist#5524] [Classification] [answered]
If I run linguist on https://github.com/WebKit/WebKit it does not show any language.

**Comment by lildude** [comment, marked as answer]
This is the same as discussed in https://github.com/github/linguist/discussions/5445. My reply from there:

> The issue here is that repo is heeeeaaawg and has a massive object tree and a lot of files so it hitting the guard at:
> 
> https://github.com/github/linguist/blob/16c70aef8cd62ca071231a380c69050f5e83c900/lib/linguist/repository.rb#L132-L135
> 
> This has been implemented so that repos like this don't negatively impact the analysis of others or excessively hog resources.
